peribolos.yaml

```diff
@@ -53,9 +53,12 @@ orgs:
         description: Emporous client libraries and CLI implementation
         has_projects: true
         homepage: https://universalreference.io/
-      emporous-docs:
+      community:
         default_branch: main
+        description: Emporous Community
+        homepage: https://universalreference.io/community
```

Comment:
Can this be updated to refer to the `emporous.io` domain?

Reply:
fixed